reporter-web-app/build.gradle

```diff
@@ -5,7 +5,11 @@ plugins {
 node {
     version = '8.11.4'
     yarnVersion = '1.9.4'
-
+    // Installing NPM is needed for installing Yarn because
```

Comment:
Commit message: Second paragraph is weird, what about: "Also set a fixed NPM version to ensure reproducible builds." Also it's a bit weird to have [2] before [1]. And nit: "Support" should be capitalized in the title for consistency with other commits.

Reply:
More commit message nits: "... install*s* Yarn ...", "... to not negatively ..."